README.md

```diff
@@ -2,13 +2,96 @@
 
 ![](Extras/img.jpg)
 
-## Python / Raspberry Pi interpretation of Bryan Boyer's Very Slow Movie Player  
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
+Very Slow Movie Player using Python + Raspberry Pi
 
-Full writeup of this project here:   
-https://medium.com/@tomwhitwell/how-to-build-a-very-slow-movie-player-in-2020-c5745052e4e4
+## Background
 
-Bryan's original post here:  
-https://medium.com/s/story/very-slow-movie-player-499f76c48b62  
+In December 2018, Bryan Boyer posted [“Creating a Very Slow Movie Player”](https://medium.com/s/story/very-slow-movie-player-499f76c48b62), an essay about light and Brasília and architecture in which Boyer builds an e-paper display that shows films at 24 frames per hour, rather than 24 frames per second so it takes about a year to play the 142 minutes of 2001: A Space Odyssey.
 
+In August 2020, Tom Whitwell posted ["How to Build a Very Slow Movie Player for £120 in 2020"](https://debugger.medium.com/how-to-build-a-very-slow-movie-player-in-2020-c5745052e4e4), with instructions on how to build a VSMP with the new [7.5-inch, Raspberry Pi-compatible e-paper display from Waveshare](https://www.waveshare.com/product/displays/e-paper/epaper-1/7.5inch-e-paper-hat.htm).
 
+SlowMovie is the code that runs a VSMP on your Raspberry Pi.
+
+## Install
+
+**Note:** These installation instructions assume you have access to your Raspberry Pi and that you have the hardware set up properly. See the [Medium post](https://debugger.medium.com/how-to-build-a-very-slow-movie-player-in-2020-c5745052e4e4) for more complete instructions.
+
+SlowMovie requires [python3](https://www.python.org/). It uses [ffmpeg](https://ffmpeg.org/) via [ffmpeg-python](https://pypi.org/project/ffmpeg-python/) for video processing, and [Pillow](https://python-pillow.org/) for image processing.
+
+On the Raspberry Pi:
+0. Make sure SPI is turned on
+  * run `sudo raspi-config`
+  * Navigate to Interfacing Options > SPI
+1. Clone this repo
+  * `git clone https://github.com/TomWhitwell/SlowMovie/`
+  * Navigate to the new SlowMovie directory: `cd SlowMovie`
+2. Install e-paper drivers
+  * Go into the e-paper driver directory: `cd e-paper/rpi/python/`
+  * Run `sudo python3 setup.py install`
```

Comment:
Pip will need to be installed before setup.py can be installed (sudo apt install python3-pip)
After that, setuptools may need to be updated (sudo pip3 install setuptools -U)

Reply:
Thanks! I'm gonna add this into #36 since that's the most up-to-date version of the readme right now.

Reply:
26c3dcf

Reply:
Now that I think about it, git isn't installed by default either. I think on NOOBS it is, but it isn't on vanilla Raspbian.

Reply:
Ooh good call. Would you mind adding bullet-points for those in #36, since you seem to have a really good handle on these dependencies? Maybe even a new step 1. called "Set up environment/dependencies" or something would be in order here. Something like:
```
0. Make sure SPI is turned on
    * ...
1. Set up environment
    * apt update you mentioned
    * git
    * Make sure pip3 is installed: `sudo apt install python3-pip`
    * Make sure setuptools is updated: `sudo pip3 install setuptools -U`
2. Install e-paper drivers
    * Clone the Waveshare repo: `git clone https://github.com/waveshare/e-Paper`
    * Go into the e-paper driver directory: `cd e-Paper/RaspberryPi_JetsonNano/python/`
    * Install the drivers: `sudo python3 setup.py install`
    * Go back out of the e-paper directory: `cd ../../..`
3. Clone this repo
    * ...
4. Make sure dependencies are installed
    * ...
5. Test it out
    * ...
```

Reply:
I merged that branch just before reading this comment, It'll have to go here.

Reply:
Thanks for typing it all out, love the improvements you made. I saw you carried them over to a commit on `master`. Sorry about the confusion there!